en/docs/get-started/about-this-release.md

```diff
@@ -398,14 +398,54 @@ The following mediators and artifacts are removed from the ESB runtime in all ve
+
+- **Improved MongoDB Store**
+    
+	 Several improvements are done like allow encryption protocol to be configurable, Object_ID support when joining and other bug fixes. For more information , see [Mongo DB Store](https://siddhi-io.github.io/siddhi-store-mongodb/api/2.1.2/#mongodb-store).
+	 
+- **Improved IO CDC Source**
+    
+	 CDC can be used to capture insert / update / delete actions from databases. Release includes improvement where with single CDC configuration it can handle all three actions. For more information , see [CDC Source](https://siddhi-io.github.io/siddhi-io-cdc/api/2.0.10/).
+	 
+- **Synchronous publishing support for IO Kafka**
+    
+	 IO Kafka allows seamless connection with kafka broker and retrieves the events into the streaming integrator. Release includes synchronous publishing in a more controlled manner and other bug fixes. For more information , see [IO-Kafka](https://github.com/siddhi-io/siddhi-io-kafka).
+	 
+- **Improved JSON Mapper to support null values**
+    
+	 JSON mapper provides capability to retrieve or publish events in JSON format. For more information , see [Map-JSON](https://github.com/siddhi-io/siddhi-map-json).
+	 
+- **Improved KeyValue Mapper to support force casting**
+    
+	 KeyValue mapper provides capability to retrieve or publish events in as key value format. For more information , see [Map-KeyValue](https://siddhi-io.github.io/siddhi-map-keyvalue/api/2.1.1/).
+	 
+- **Improved IO File Sink and Source**
+    
+	 IO File sink / source can be used to read or write into a file. With the new versions it has the support for EOF in other modes, support for file trailers, file writing mechanisms and SFTP support. For more information , see [IO-File](https://siddhi-io.github.io/siddhi-io-file/api/2.0.16/).
+	 
+- **Improved Store RDBMS support**
+    
+	 Store RDBMS can be used to connect with any rdbms type. With the new versions it has the support to add null values to RDBMS tables and improved metrics support. For more information , see [Store RDBMS](https://github.com/siddhi-io/siddhi-store-rdbms).
+	 
+- **Added Kafka and HTTP metrics monitoring capabilities**
+    
+	 This release provides Kafka and HTTP metrics monitoring capability. In the previous release, this was only available for File, RDBMS and CDC. The metrics can be published to a pre-configured [Prometheus](https://prometheus.io/) server and then you can either create your own [Grafana](https://grafana.com/) dashboard or use the ETL dashboard which is already created by us. For more information on how to configure metrics, refer to [Monitoring ETL Flows](https://apim.docs.wso2.com/en/4.0.0/observe/si-observe/viewing-dashboards/) documentation page.
```

Comment:
The description says "previously, these features were available for certain scenarios..". So, what's new now? Improved capabilities?
Suggestion:
"This Streaming Integrator release provides improved Kafka and HTTP metrics monitoring capabilities. Previously, Kafka and HTTP metrics monitoring capabilities were only available for File, RDBMS, and CDC. You can now publish the metrics to a pre-configured [Prometheus](https://prometheus.io/) server and then you can either create your own [Grafana](https://grafana.com/) dashboard or use the ETL dashboard which is already available with the Streaming Integrator. For more information on how to configure metrics, see [Monitoring ETL Flows](https://apim.docs.wso2.com/en/4.0.0/observe/si-observe/viewing-dashboards/) documentation page."

Reply:
No suggestion need to change. as per the topic what newly added was Kafka and HTTP metrics monitoring capabilities. Earlier we had the monitoring capabilities only for  File, RDBMS, and CDC. So I guess this is fine. WDYT ?